src/coreclr/tools/Common/JitInterface/CorInfoImpl.cs

```diff
@@ -3252,6 +3269,7 @@ private void allocMem(ref AllocMemArgs args)
                 args.coldCodeBlock = (void*)GetPin(_coldCode = new byte[args.coldCodeSize]);
                 args.coldCodeBlockRW = args.coldCodeBlock;
             }
+            allocatedHotCodeSize = args.hotCodeSize;
```

Comment:
Can we just use `_code.Length` instead of introducing this field?

Reply:
Oh yes, I used it during logging but forgot to get rid of that field.